src/libraries/Microsoft.Extensions.Configuration.Binder/src/ConfigurationBinder.cs

```diff
@@ -79,11 +79,11 @@ public static object Get(this IConfiguration configuration, Type type)
         /// <param name="configureOptions">Configures the binder options.</param>
         /// <returns>The new instance if successful, null otherwise.</returns>
         [RequiresUnreferencedCode(TrimmingWarningMessage)]
-        public static object Get(
+        public static object? Get(
             this IConfiguration configuration,
             [DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.All)]
             Type type,
-            Action<BinderOptions> configureOptions)
+            Action<BinderOptions>? configureOptions)
         {
             if (configuration == null)
```

Comment:
Maybe it's better to remove null-check here? It would be easier to call `Get` with `null` than with `_ => { }`